mypy/test/testcheck.py

```diff
@@ -147,15 +147,17 @@ def run_case_once(self, testcase: DataDrivenTestCase, incremental_step: int = 0)
                         if file.endswith('.' + str(incremental_step)):
                             full = os.path.join(dn, file)
                             target = full[:-2]
-                            shutil.copy(full, target)
+                            # Use retries to work around potential flakiness on Windows (AppVeyor).
+                            retry_on_error(lambda: shutil.copy(full, target))
 
                             # In some systems, mtime has a resolution of 1 second which can cause
                             # annoying-to-debug issues when a file has the same size after a
                             # change. We manually set the mtime to circumvent this.
                             new_time = os.stat(target).st_mtime + 1
                             os.utime(target, times=(new_time, new_time))
                 for path in testcase.deleted_paths.get(incremental_step, set()):
-                    os.remove(path)
+                    # Use retries to work around potential flakiness on Windows (AppVeyor).
+                    retry_on_error(lambda: os.remove(path))
```

Comment:
Changed the return type to `Any`. The type check might be worth filing as a mypy bug, since it looks like mypy should be able to infer the call -- I'll think about it.